src/components/confirm/confirm.spec.js

```diff
@@ -98,8 +100,41 @@ describe("Confirm", () => {
       });
     });
 
+    it("should not render confirm button", () => {
+      wrapper = mount(<Confirm onConfirm={() => {}} isLoadingConfirm open />);
+
+      expect(wrapper.find(Loader).exists()).toBe(true);
+    });
+
+    it("should render confirm button without left margin", () => {
```

Comment:
Now you've asserted the `marginLeft`, you might want to change the title of this test to something more appropriate.

Reply:
Oh you are right 🤦‍♂️